staging/src/k8s.io/apiserver/pkg/endpoints/responsewriter/wrapper_test.go

```diff
@@ -29,108 +29,68 @@ import (
 	responsewritertesting "k8s.io/apiserver/pkg/endpoints/responsewriter/testing"
 )
 
```

Comment:
this commit does not introduce any change in the test, it just merges the two tests into one.